app/src/test/java/de/westnordost/streetcomplete/quests/existence/CheckExistenceTest.kt

```diff
@@ -36,4 +38,61 @@ class CheckExistenceTest {
             StringMapEntryDelete("survey_date", "d"),
         )
     }
+
+    @Test fun `title is constructed correctly, issue #2512`() {
+        val newspaperVendingMachineWithName = questType.getQuestTitle(mapOf(
+            "amenity" to "vending_machine",
+            "vending" to "newspapers",
+            "name" to "Bild",
+        ))
+        assertEquals(newspaperVendingMachineWithName, "Is Bild (vending machine) still here?")
```

Comment:
How can this work? This is a `test`, not an `androidTest` right? So how is it possible that `de.westnordost.streetcomplete.R` is available?

If this works... then I don't know how, and I think it would be better to move these tests to `androidTests`